packages/nodes-base/credentials/ZoomOAuth2Api.credentials.ts

```diff
@@ -0,0 +1,50 @@
+import { ICredentialType, NodePropertyTypes } from 'n8n-workflow';
+
+const userScopes = [
+	'meeting:read',
+	'meeting:write',
+	'user:read',
+	'user:write',
+	'user_profile',
+	'webinar:read',
+	'webinar:write'
+];
```

Comment:
You are not using `userScopes`?

Reply:
``` javascript
properties = [
    {
		displayName: 'Scope',
		name: 'scope',
		type: 'hidden' as NodePropertyTypes,
		default: scopes.join(' ')
    },
];
```

This will also prevent the error I encountered as a user when I did not set the scopes!